python/lancedb/embeddings.py

```diff
@@ -64,13 +66,17 @@ def embed_func(c):
                 return self.func(c.tolist())
 
         if len(self.rate_limiter_kwargs) > 0:
-            import ratelimiter
-
-            max_calls = self.rate_limiter_kwargs["max_calls"]
-            limiter = ratelimiter.RateLimiter(
-                max_calls, period=self.rate_limiter_kwargs["period"]
-            )
-            embed_func = limiter(embed_func)
+            v = int(sys.version_info.minor)
+            if v >= 11:
```

Comment:
will `ratelimiter` throw an execption or something? if so, just catch the exception?